pkgs/by-name/vc/vcsh/package.nix

```diff
@@ -2,24 +2,30 @@
   lib,
   stdenv,
   fetchurl,
+  autoconf,
+  automake,
   makeWrapper,
   pkg-config,
+  unzip,
   git,
   perlPackages,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "vcsh";
-  version = "2.0.8";
+  version = "2.0.10";
 
   src = fetchurl {
-    url = "https://github.com/RichiH/vcsh/releases/download/v${version}/${pname}-${version}.tar.xz";
-    sha256 = "sha256-VgRA3v5PIKwizmXoc8f/YMoMCDGFJK/m2uhq3EsT1xQ=";
+    url = "https://github.com/RichiH/vcsh/releases/download/v${finalAttrs.version}/vcsh-${finalAttrs.version}.zip";
```

Comment:
If we're already pulling in autotools now, we should switch to `fetchFromGitHub`. This would only need `ronn` as build time dependency in addition, which we already have in nixpkgs.

https://github.com/RichiH/vcsh/blob/main/doc/INSTALL.md#installing-from-source